branch_test/demo_query_method.py

```diff
@@ -0,0 +1,271 @@
+import os
```

Comment:
`branch_test` should be removed...

Reply:
I will remove it once it is tested and approved by the clinical team. It helps debugging of newly created modules.